kore/src/Kore/Log/Registry.hs

```diff
@@ -139,8 +139,8 @@ import Kore.Log.JsonRpc (
 import Kore.Log.WarnBoundedModelChecker (
     WarnBoundedModelChecker,
  )
-import Kore.Log.WarnClaimRHSIsBottom (
-    WarnClaimRHSIsBottom,
+import Kore.Log.WarnBottom (
+    WarnClaimRHSIsBottom, WarnConfigIsBottom
```

Comment:
Same here, imports not sorted alphabetically. I thought the formatter would take care of that.

Reply:
Oh, yes, the formatter hasn't run yet